### PR TITLE
[Feature] Enhance job submission frontend

### PR DESCRIPTION
**Pull request by s7monk**
close: https://github.com/apache/paimon-webui/issues/265

### Purpose

 Supports job submit frontend,completed job submission and result display.

image

image

image

image

### CODE REVIEW

File: paimon-web-ui/src/views/playground/components/query/components/console/components/controls/index.tsx [hunk 1]
```diff
@@ -16,29 +16,227 @@ specific language governing permissions and limitations
...
+    const handleRefreshData = async () => {
+      if (currentJob.value) {
+        if (currentJob.value.shouldFetchResult) {
+          try {
+            const resultFetchDTO = {
+              submitId: currentJob.value.submitId,
+              clusterId: currentJob.value.clusterId,
+              sessionId: currentJob.value.sessionId,
+              taskType: currentJob.value.type,
+              token: currentJob.value.token
```
Comment (zhangmo8) [inline paimon-web-ui/src/views/playground/components/query/components/console/components/controls/index.tsx:60]: Can be use the Extended symbol. Remember to use toRaw before use

File: paimon-web-ui/src/views/playground/components/query/components/console/components/controls/index.tsx [hunk 2]
```diff
@@ -16,29 +16,227 @@ specific language governing permissions and limitations
...
+    const handleStopJob = async () => {
+      if (currentJob.value) {
+        const stopJobDTO = {
+          clusterId: currentJob.value.clusterId,
+          jobId: currentJob.value.jobId,
+          taskType: currentJob.value.type,
+          withSavepoint: false
+        }
+        try {
+          const response: any = await stopJob(stopJobDTO);
```
Comment (zhangmo8) [inline paimon-web-ui/src/views/playground/components/query/components/console/components/controls/index.tsx:88]: Please use the correct type

File: paimon-web-ui/src/views/playground/components/query/components/console/components/controls/index.tsx [hunk 3]
```diff
@@ -16,29 +16,227 @@ specific language governing permissions and limitations
...
+          message.warning(t('playground.job_stopping_failed'))
+        }
+      }
+    }
+
+    let getJobStatusIntervalId: number
+    onMounted(() => {
+      getJobStatusIntervalId = setInterval(async () => {
+        if (currentJob.value && currentJob.value.jobId) {
+          const response: any = await getJobStatus(currentJob.value.jobId)
```
Comment (zhangmo8) [inline paimon-web-ui/src/views/playground/components/query/components/console/components/controls/index.tsx:104]: Please use the correct type

File: paimon-web-ui/src/views/playground/components/query/components/console/components/controls/index.tsx [hunk 4]
```diff
@@ -16,29 +16,227 @@ specific language governing permissions and limitations
...
+        case '1m':
+          setRefreshInterval(60000)
+          break
+        case '5m':
+          setRefreshInterval(300000)
+          break
+        case 'Disabled':
+        default:
+          clearRefreshInterval()
+          break
```
Comment (zhangmo8) [inline paimon-web-ui/src/views/playground/components/query/components/console/components/controls/index.tsx:179]: Please use dayjs for time conversion

File: paimon-web-ui/src/locales/zh/modules/playground.ts [hunk 5]
```diff
@@ -49,4 +49,10 @@ export default {
   schedule_refresh: '定时刷新',
   download: '导出数据',
   copy: '复制数据',
+  job_submission_successfully: 'SQL 任务提交成功！',
```
Comment (SteNicholas) [inline paimon-web-ui/src/locales/zh/modules/playground.ts:52]: Remove “!”.

**Review by zhangmo8 — APPROVED** [review]
LGTM

**Review by SteNicholas — APPROVED** [review]

**Pull request merged**